<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMSLogDialog</class>
 <widget class="QDialog" name="SMSLogDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SMS Logs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="deviceLabel">
     <property name="text">
      <string>SMS Logs for Device</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="smsTable"/>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
